src/hooks/auditorium/useAllAuditoriumSections.ts

```diff
@@ -62,16 +58,10 @@ export const useAllAuditoriumSections = (venue: WithId<AuditoriumVenue>) => {
 
     return sections.filter((section) => {
       const sectionCapacity = getSectionCapacity(venue, section);
-      const seatedUsers = getAuditoriumSeatedUsers({
-        venueId,
-        // @debt should be replaced with a subcollection
-        auditoriumUsers: recentVenueUsers,
-        sectionId: section.id,
-      });
 
-      return seatedUsers.length < sectionCapacity;
+      return (section?.seatedUsersCount ?? 0) < sectionCapacity;
```

Comment:
move seatedUsersCount to a var

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2318/commits/10fe3f04176fbe949d0025a18c5ea6761b3b44d8